src/pages/PointOfContact/index.tsx

```diff
@@ -256,7 +256,7 @@ function PointOfContact(): JSX.Element {
             You are required to complete all fields with the exception of the
             street address lines labeled optional. Your point of contact
             information will not be saved until you provide all required
-            information and click &quot;Save and continue.&quot;
+            information and click &quot;Continue to next step.&quot;
```

Comment:
Update: 
'You are required to complete all fields with the exception of the street address lines labeled optional. Your point of contact information will not be saved until you provide all required information and continue to the next step.'

Reply:
Heyo @natalia-fitzgerald, these are handled in my PR over here:
https://github.com/cfpb/sbl-frontend/pull/855/files